docs/releases/1.11.1.txt

```diff
@@ -9,4 +9,6 @@ Django 1.11.1 fixes several bugs in 1.11.
 Bugfixes
 ========
 
-* ...
+* Django 1.11 inadvertently set the name for an index during migrations even if
+  one was explicitly set. The fix will result rebuilding those indexes.
```

Comment:
Might be better as  `If you have created an explicitly named index with django 1.11, a new migration file will be created which will rebuild the index.`

Or at least `will result in rebuilding` is needed to read correctly